backend/server.js

```diff
@@ -1,29 +1,177 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/authAPI"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import mongoose from "mongoose";
+import crypto from "crypto";
+import bcrypt from "bcrypt-nodejs";
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/authAPI";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+    minlength: 5,
+  },
+  password: {
+    type: String,
+    required: true,
+    minlength: 8,
+  },
+  accessToken: {
+    type: String,
+    unique: true,
```

Comment:
You need this line here: 
` default: () => crypto.randomBytes(128).toString('hex')`

Reply:
Added this. Also removed unique required to fix the server not working. I'm setting the token to null when logging out so it can't be unique. Now everything is working fine:)